src/poset/poset.go

```diff
@@ -1243,22 +1288,22 @@ func (p *Poset) ProcessDecidedRounds() error {
 		//Indeed, after a Reset, LastConsensusRound is added to PendingRounds,
 		//but its ConsensusEvents (which are necessarily 'under' this Round) are
 		//already deemed committed. Hence, skip this Round after a Reset.
-		if r.Index == p.GetLastConsensusRound() {
+		if r == p.GetLastConsensusRound() {
 			continue
 		}
 
-		frame, err := p.GetFrame(r.Index)
+		frame, err := p.GetFrame(r)
 		if err != nil {
-			return fmt.Errorf("getting Frame %d: %v", r.Index, err)
+			return fmt.Errorf("getting Frame %d: %v", r, err)
 		}
 
-		round, err := p.Store.GetRound(r.Index)
-		if err != nil {
-			return err
-		}
+		//round, err := p.Store.GetRoundReceived(r)
```

Comment:
Use space please. https://github.com/golang/go/wiki/Comments
```
// round, err := p.Store.GetRoundReceived(r)
```
In the place and in other places.

Reply:
Will do